tests/tests_pytorch/utilities/test_data.py

```diff
@@ -328,7 +330,118 @@ def test_replace_dataloader_init_method(cls, args, kwargs, arg_names, dataset, c
         if isinstance(dataloader_value, torch.Tensor):
             assert dataloader_value is value
         else:
-            assert getattr(dataloader, key) == value
+            assert dataloader_value == value
+
+
+def test_replace_init_method_extra_kwargs():
```

Comment:
Was this case not covered by the existing dataloader tests?

Reply:
Surprisingly no. And they way existing tests were set up, it couldn't even be covered, because it checked whether `__pl_saved_kwargs == kwargs`, which were the ones passed to the test. So it did not do anything with the default kwargs at all